test/hotspot/jtreg/runtime/SharedArchiveFile/MaxMetaspaceSize.java

```diff
@@ -45,12 +47,11 @@ public static void main(String[] args) throws Exception {
     if (Platform.is64bit()) {
       processArgs.add("-XX:MaxMetaspaceSize=3m");
       processArgs.add("-XX:CompressedClassSpaceSize=1m");
-      processArgs.add("-XX:InitialBootClassLoaderMetaspaceSize=1m");
     } else {
```

Comment:
Includes the changes from [8251158](https://bugs.openjdk.org/browse/JDK-8251158) on this file